build.sbt

```diff
@@ -13,15 +13,18 @@ lazy val suffuse = ( project in file(".")
 
 lazy val core = project dependsOn `fuse-jna` settings common
 
-def common = Seq[Setting[_]](
-                organization :=  "org.improving",
-    scalacOptions in compile ++= Seq("-Ywarn-unused", "-Ywarn-unused-import"),
-         libraryDependencies +=  "com.novocode" % "junit-interface" % "0.11" % "test",
-                    licenses :=  Seq("Apache-2.0" -> url("http://www.apache.org/licenses/LICENSE-2.0")),
-                scalaVersion :=  "2.11.8",
-                 logBuffered :=  false,
-                   maxErrors :=  15,
-            triggeredMessage :=  Watched.clearWhenTriggered
+def common: Seq[Setting[_]] = Def settings (
+                           organization :=  "org.improving",
+                          scalacOptions ++= Seq("-feature", "-language:_", "-unchecked"),
+        inBoth(scalacOptions in compile ++= Seq("-Ywarn-unused", "-Ywarn-unused-import")),
```

Comment:
Why should this be in both? Will it not compile before it tests?

Maybe the `in compile` should be `in Compile`. You probably know better than I do, haha :smile: